README.md

```diff
@@ -150,41 +150,12 @@ $ ocic pod run --config test/testdata/sandbox_config.json
 
 ### Setup CNI networking
 
-Follow the steps below in order to setup networking in your pods using the CNI
-bridge plugin. Nothing else is required after this since `CRI-O` automatically
-setup networking if it finds any CNI plugin.
-
-```sh
-$ go get -d github.com/containernetworking/cni
-$ cd $GOPATH/src/github.com/containernetworking/cni
-$ sudo mkdir -p /etc/cni/net.d
-$ sudo sh -c 'cat >/etc/cni/net.d/10-mynet.conf <<-EOF
-{
-    "cniVersion": "0.2.0",
-    "name": "mynet",
-    "type": "bridge",
-    "bridge": "cni0",
-    "isGateway": true,
-    "ipMasq": true,
-    "ipam": {
-        "type": "host-local",
-        "subnet": "10.88.0.0/16",
-        "routes": [
-            { "dst": "0.0.0.0/0"  }
-        ]
-    }
-}
-EOF'
-$ sudo sh -c 'cat >/etc/cni/net.d/99-loopback.conf <<-EOF
-{
-    "cniVersion": "0.2.0",
-    "type": "loopback"
-}
-EOF'
-$ ./build
-$ sudo mkdir -p /opt/cni/bin
-$ sudo cp bin/* /opt/cni/bin/
-```
+In order to set up CNI networking in your pods, you need to have CNI networks
+defined in `/etc/cni/net.d`. There are examples of such configurations
+available in `contrib/cni`, which will set up a simple bridge
```

Comment:
nit: add a link of `contrib/cni`, e.g. [contrib/cni](https://github.com/kubernetes-incubator/cri-o/tree/master/contrib/cni)